sycl/test/format.py

```diff
@@ -21,7 +23,8 @@ def getTestsInDirectory(self, testSuite, path_in_suite, litConfig, localConfig):
         for dirpath, _, filenames in os.walk(source_path):
             relative_dirpath = dirpath[len(localConfig.sycl_include) + 1 :]
             for filename in filenames:
-                if not filename.endswith(".hpp"):
+                suffix = os.path.splitext(filename)[1]
+                if suffix not in SUFFIXES or suffix not in litConfig.suffixes:
```

Comment:
@ldrumm, I've just realized that this patch had completely disable the testing suite :)

We don't have `.hpp` registered as a suffix and therefore we always skip all those tests. This is more of a FYI, I'm working on a revamp/expansion of the suite in scope of #16012, so I will fix it as part of that effort